.github/ISSUE_TEMPLATE/backup-support.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
I think this template already exists as `support-request.md` - can you edit that file vs making this new one? I don't want to confuse users by having too many support templates.